test/dlc_tests/sanity/quick_checks/test_dlc_developer_config.py

```diff
@@ -19,6 +19,7 @@ def test_developer_configuration():
     assert config.parse_dlc_developer_configs("dev", "benchmark_mode") is False
     assert config.parse_dlc_developer_configs("dev", "habana_mode") is False
     assert config.parse_dlc_developer_configs("dev", "trcomp_mode") is False
+    assert config.parse_dlc_developer_configs("dev", "deep_canary_mode") is False
```

Comment:
mentioned above, but it may be useful to test some of the new config parsing functions you've introduced above